UI/frontend-plugins/frontend-tools/data/scripts/instant-replay.lua

```diff
@@ -106,14 +128,26 @@ function script_properties()
 			if source_id == "ffmpeg_source" then
 				local name = obs.obs_source_get_name(source)
 				obs.obs_property_list_add_string(p, name, name)
+			elseif source_id == "vlc_source" then
+				local name = obs.obs_source_get_name(source)
+				obs.obs_property_list_add_string(p, name, name)
+			else
+				-- obs.script_log(obs.LOG_INFO, source_id)
 			end
 		end
 	end
 	obs.source_list_release(sources)
-
```

Comment:
Keep this blank line.  You might also want to add a blank line before the `return props` line.